Dockerfile

```diff
@@ -56,7 +56,7 @@ ENV CRATE_HEAP_SIZE 512M
 # if we cannot get any response from the CrateDB (connection refused, timeout
 # etc). If any response is received (regardless of http status code) we
 # consider the node as running.
-HEALTHCHECK --timeout=30s --interval=30s CMD curl --max-time 25 $(hostname):4200 || exit 1
+HEALTHCHECK --timeout=30s --interval=30s CMD curl --max-time 12 localhost:4200 || curl --max-time 12 https://localhost:4200 --insecure || exit 1
```

Comment:
Do we need the healthcheck chain, or would it also work if the healthcheck always uses the `--insecure` flag?

Reply:
I think we need the healthcheck chain - in an SSL enabled setup curling against `localhost:4200` even with ``--insecure`` returns an empty response from the server. Even if the node was working properly, the healthcheck would think it has failed.